test/JDBC/expected/BABEL-3317.out

```diff
@@ -0,0 +1,44 @@
+SET PARSEONLY ON;
+go
+CREATE TABLE t1(a int);
```

Comment:
Let's use `babel_3317_t1`. General table names such as `t1` can cause a problem when we are doing parallel tests.